README.rst

```diff
@@ -12,7 +63,7 @@ applications to use it.  Start off by looking at the `examples`_,
 
 
 Quick Example
-===============
+=============
```

Comment:
IMO this example only serves to attract new uses, so I'd suggest to drop it completely, too.